docs/blog/0.54.md

```diff
@@ -0,0 +1,30 @@
+---
+
+date: 2025-01-30
+image: https://github.com/rilldata/rill/assets/5587788/b30486f6-002a-445d-8a1b-955b6ec0066d
+
+---
+
+# Rill 0.54 - Bug fixes, pushing envs on deploy and many small improvements
+
+:::note
+⚡ Rill Developer is a tool that makes it effortless to transform your datasets with SQL and create fast, exploratory dashboards. Rill Cloud can then help to enable shared collaboration at scale.
+
+To [try out Rill Developer, check out these instructions](/home/install) and [join us over on Discord](https://bit.ly/3bbcSl9) to meet the team behind the product as well as other users. In addition, once you have a project set up in Rill Developer, you can then [deploy the project](/deploy/deploy-dashboard) and collaborate with others / invite your team members by [logging into Rill Cloud](https://ui.rilldata.com)!
+:::
+
+## Pushing envs on deploy
+Rill Developer now ensures that any local .envs are automatically pushed whenever you rill deploy to ensure that no credentials are left behind!
+
+## Bug fixes and small improvements
+Many bug fixes and small improvements went into this release as well as a lot of work into features coming in upcoming release so stay tuned for 0.55 and beyond!
```

Comment:
Missing a comma: "went into this release, as well as..."

Missing an article: "coming in _the_ upcoming release"

Missing a comma: "upcoming release, so stay tuned..."